ProjectTemplates/configuration.json

```diff
@@ -11,6 +11,9 @@
         "B2C_WebApi_ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
         "B2C_WebApi_Port": "44332",
 
+        "B2C_Blazorwasm_ClientId": "25f7fc07-bb5b-49ad-8179-6d486d094ca6",
```

Comment:
This is to make it simpler to test the projects generated from the B2C blazor templates (because our test apps would require to change the route in the controller, etc ...). Using a new client makes it easier